csharp/private/repositories.bzl

```diff
@@ -98,6 +98,391 @@ def csharp_repositories():
         sha256 = "912f6eed993e77c83cb8b92db72d50df7d06e4c4a02486474eae460728291989",
     )
 
+	nuget_package(
+		name = "netstandard2plus"
+		package = "NetStandard.Library"
+		version = "2.0.3"
+		sha256 = "3eb87644f79bcffb3c0331dbdac3c7837265f2cdf58a7bfd93e431776f77c9ba"
+	)
+
+    nuget_package(
+        name = "System.Collections"
+        package = "System.Collections"
```

Comment:
The rest of these are what `NETStandard.Library` depends on (directly or transitively). They each have varying targeted frameworks available in `ref/`.

Reply:
Some of these may not fully apply to netstandard, and only be here for netcore, but I didn't necessarily get every package netcore needs.